### ISSUE TITLE
Does not work for YesWeHack (ywh)

### ISSUE THREAD

**Issue by cosad3s** [sw33tLie/bbscope#35]
Hello,

I have tested bbscope with latest version and it seems it does not work anymore with YWH:

```bash
❯ ./bbscope ywh -t eyJ[REDACTED]
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/sw33tLie/bbscope/pkg/platforms/yeswehack.GetProgramScope({0x7ffcda25aaf7, 0x383}, {0xc0000fa582?, 0x0?}, {0x8aa175, 0x3})
        /home/seb/go/pkg/mod/github.com/sw33t!lie/bbscope@v0.0.0-20230704144330-81056a6c1d1d/pkg/platforms/yeswehack/yeswehack.go:69 +0x7a5
github.com/sw33tLie/bbscope/pkg/platforms/yeswehack.GetAllProgramsScope({0x7ffcda25aaf7, 0x383}, 0x0, 0x0, {0x8aa175, 0x3})
        /home/seb/go/pkg/mod/github.com/sw33t!lie/bbscope@v0.0.0-20230704144330-81056a6c1d1d/pkg/platforms/yeswehack/yeswehack.go:107 +0x84f
github.com/sw33tLie/bbscope/pkg/platforms/yeswehack.PrintAllScope({0x7ffcda25aaf7?, 0xc000187d70?}, 0x10?, 0x7d?, {0x8aa175?, 0xc0001b2460?}, {0x95f258, 0x1}, {0x960b70, 0x1})
        /home/seb/go/pkg/mod/github.com/sw33t!lie/bbscope@v0.0.0-20230704144330-81056a6c1d1d/pkg/platforms/yeswehack/yeswehack.go:121 +0x4b
github.com/sw33tLie/bbscope/cmd.glob..func5(0xbffc00?, {0x8aa743?, 0x2?, 0x2?})
        /home/seb/go/pkg/mod/github.com/sw33t!lie/bbscope@v0.0.0-20230704144330-81056a6c1d1d/cmd/ywh.go:38 +0x2f0
github.com/spf13/cobra.(*Command).execute(0xbffc00, {0xc000134480, 0x2, 0x2})
        /home/seb/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:860 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0xbfef80)
        /home/seb/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /home/seb/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/sw33tLie/bbscope/cmd.Execute()
        /home/seb/go/pkg/mod/github.com/sw33t!lie/bbscope@v0.0.0-20230704144330-81056a6c1d1d/cmd/root.go:28 +0x25
main.main()
        /home/seb/go/pkg/mod/github.com/sw33t!lie/bbscope@v0.0.0-20230704144330-81056a6c1d1d/main.go:6 +0x17
```

**Comment by cosad3s** [comment]
See: https://github.com/sw33tLie/bbscope/pull/36

**Comment by sw33tLie** [comment]
Fixed in #36
